src/app/pipes/linkColumn.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'linkColumn'
+})
+export class LinkColumnPipe implements PipeTransform {
```

Comment:
How about renaming this to `LinkColumnTypePipe` and just have this be responsible for stating what type of link this is?